webpack.config.js

```diff
@@ -86,11 +93,17 @@ module.exports = {
         ]
     },
     plugins: [
-        new CopyPlugin({
-          patterns: [{
-            from: "**/*",
-            context: path.resolve(__dirname, "src", "public")
-            }]
```

Comment:
It is the right idea to try to remove this because it will not generally work with the new deployment mechanism. However, there are a lot of files in `src/public` that will no longer be copied with this removed. 
Specifically there are lots of html files in `src/public/examples`. You can see an old version of one of these deployed here: https://sensor-interactive.concord.org/examples/fake-sensor.html

We don't have a great pattern for supporting these kind of auxiliary html files in the new deployment system. The release_production.yml and release_staging.yml only copy the index-top.html file by default. `release_production.yml` could be changed to copy the full examples folder. However often this will fail because these kinds of html files usually assume they'll be able to find the javascript files at location which won't be in the same place after the file is copied.  But in this case I see things are setup so the example javascript files added to the dist/example folder too. So the html files reference them with a relative path. For example see this source: view-source:https://sensor-interactive.concord.org/branch/master/examples/fake-sensor-bar-prediction-prerecorded.html 

So in this case it would be safe to update the `release_production.yml` to copy all of the files in the examples folder of the version being deployed up to the production examples folder.  There should be a `aws s3 cp` command you can construct to copy a whole folder of files.

Based on the S3 files in s3://models-resources/sensor-interactive/examples/ the examples haven't been updated on production in a long time. There are only 5 files in there compared with 72 files in s3://models-resources/sensor-interactive/branch/master/examples/